Makefile

```diff
@@ -79,6 +79,9 @@ cross: build ## cross build the binaries for darwin, freebsd and\nwindows
 win: build ## cross build the binary for windows
 	$(DOCKER_RUN_DOCKER) hack/make.sh win
 
+darwin: build ## cross build the binary for OS X
+	$(DOCKER_RUN_DOCKER) hack/make.sh darwin
```

Comment:
It was just a convenience at the time. I didn't know of another way to build for windows and I was spending a long time building windows. The only way to build windows was to build cross  and it takes a long time.